src/main/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/UserEndpoints.scala

```diff
@@ -33,7 +64,7 @@ class UserEndpoints[F[_]: Effect] extends Http4sDsl[F] {
         }
     }
 
-  private def updateEndpoint(userService: UserService[F]): HttpService[F] =
+  private def updateEndpoint(userService: UserService[F], Auth: AuthService): HttpService[F] =
```

Comment:
I take it that all these endpoints actually have to do something with `Auth: AuthService` at some point?